guides/micronaut-liquibase/micronaut-liquibase.adoc

```diff
@@ -0,0 +1,193 @@
+include::{commondir}/common-header.adoc[]
+
+You are going to use https://docs.oracle.com/en/java/javase/16/language/records.html[Record Classes] in a Micronaut Application.
+
+____
+Record classes, which are a special kind of class, help to model plain data aggregates with less ceremony than normal classes.
+
+A record declaration specifies in a header a description of its contents; the appropriate accessors, constructor, equals, hashCode, and toString methods are created automatically. A record's fields are final because the class is intended to serve as a simple "data carrier".
+____
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-app
+   example.micronaut.micronautguide \
+   --features=data-jdbc,postgres,liquibase \ 
+   --build=@build@ \
+   --lang=@lang@ \
+----
+
+
+NOTE: To use Java records, specify JDK 17 when you create the application
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+include::{commondir}/common-default-package.adoc[]
+
+NOTE: If you use https://launch.micronaut.io[Micronaut Launch], select "Micronaut Application" as application type and add the `postgres`, `data-jdbc` and `liquibase` as features.
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+=== Create Entity
+
+Create a `@MappedEntity` to save persons. Initially, consider name and age required. Use the `int` primitive for the age.
+
+source:Person[tags=clazz|ageint|gettersetters]
+
+callout:mapped-entity[1]
```

Comment:
This callout is rendered as: `Annotate the class with @MappedEntity to map to the book table defined in the schema to this entity.` but you are using `Person` not `Book`